### ISSUE TITLE
Support RTL for amp-nested-menu

### ISSUE THREAD

**Issue by leafsy** [ampproject/amphtml#25157]
`amp-nested-menu` should behave reasonably for RTL documents. This includes flipping the open/close animation so that submenu enters from the left rather than right, as well as using right arrow key instead of left to navigate back.
